superbench/monitor/monitor.py

```diff
@@ -0,0 +1,261 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""Module of the Monitor class."""
+
+import os
+import time
+import glob
+import sched
+import multiprocessing
+
+from superbench.common.utils import logger, run_command
+from superbench.common.utils import device_manager as dm
+from superbench.monitor.record import MonitorRecord
+
+
+class Monitor(multiprocessing.Process):
```

Comment:
Just one general comment: We don't have child class of Monitor, therefore, it may change to __ for variable and function